src/notation/notationerrors.h

```diff
@@ -129,6 +127,7 @@ inline Ret make_ret(Err err, const io::path& filePath = "")
     case Err::UnknownError:
     case Err::UserAbort:
     case Err::IgnoreError:
+    case Err::EmptySelection:
```

Comment:
Do we want to ignore this (new) 'error'?

Reply:
better fix it as in this commit https://github.com/musescore/MuseScore/pull/10273/commits/e0c96de2e77ac9a6c8222b92606c108da7fb306d

Reply:
OK, done